exercises/practice/scale-generator/.docs/instructions.md

```diff
@@ -1,37 +1,48 @@
 # Instructions
 
-Given a tonic, or starting note, and a set of intervals, generate
-the musical scale starting with the tonic and following the
-specified interval pattern.
+## Chromatic Scales
 
 Scales in Western music are based on the chromatic (12-note) scale. This
 scale can be expressed as the following group of pitches:
 
-A, A#, B, C, C#, D, D#, E, F, F#, G, G#
+> A, A#, B, C, C#, D, D#, E, F, F#, G, G#
```

Comment:
Should these change to actual utf sharp characters (U+266F, or maybe there's a combining superscript?) instead of hash/pound symbols?

Reply:
🤷 Maybe open an issue (or a PR) at https://github.com/exercism/problem-specifications?

Reply:
Whoops, forgot this was the C# repo!